crypto/fipsmodule/aes/asm/aesni-x86.pl

```diff
@@ -153,17 +153,19 @@ sub aesni_generate1	# fully unrolled loop
 	&cmp		($rounds,11);
 	&jb		(&label("${p}128"));
 	&lea		($key,&DWP(0x20,$key));
-	&je		(&label("${p}192"));
-	&lea		($key,&DWP(0x20,$key));
+	# 192-bit key support was removed
+
+	# 14 rounds
```

Comment:
I would prefer to not add this comment. They're helpful but they'd be better added upstream and then imported through the merge process.

Reply:
Do you mean the "14 rounds/10rounds" comments,  the "192-bit key comment" or both?